composeml/label_maker.py

```diff
@@ -14,23 +14,23 @@ class LabelMaker:
 
     def __init__(
         self,
-        target_dataframe_name,
+        target_dataframe_index,
         time_index,
         labeling_function=None,
         window_size=None,
     ):
         """Creates an instance of label maker.
 
         Args:
-            target_dataframe_name (str): Dataframe on which to make labels.
+            target_dataframe_index (str): Column on which to make labels.
```

Comment:
`Column` -> `Index of the target dataframe`?

Reply:
Fixed: https://github.com/alteryx/compose/pull/353/commits/2f817c80ead4acefbc6b8cd886c7916e84170e84